app/shared/main-navbar/MainNavbar.js

```diff
@@ -65,6 +65,13 @@ class MainNavbar extends React.Component {
                 </NavItem>
               </LinkContainer>
             )}
+            {isLoggedIn && (
+              <LinkContainer to="/manage-reservations">
+                <NavItem eventKey="manage-reservations" onClick={() => this.collapseItem()}>
+                  {t('Navbar.manageReservations')}
+                </NavItem>
+              </LinkContainer>
+            )}
```

Comment:
If you can, please do some dummy test ( can copy from navbar test, they're pretty similar )